templates/verify.txt

```diff
@@ -1,6 +1,6 @@
-{{t "Almost there"}}
+{{t "Welcome!"}}
```

Comment:
I don't recall the exclamation marks being called into question but **Congratulations** was a definite trigger.